src/test/resources/phenotype/phenotype.omop.json

```diff
@@ -2,6 +2,6 @@
+    "expression": "{\"Title\":\"PhenotypeCase\",\"PrimaryCriteria\":{\"CriteriaList\":[{\"VisitOccurrence\":{\"VisitTypeExclude\":false}}],\"ObservationWindow\":{\"PriorDays\":0,\"PostDays\":0},\"PrimaryCriteriaLimit\":{\"Type\":\"First\"}},\"ConceptSets\":[{\"id\":0,\"name\":\"Diabetes\",\"expression\":{\"items\":[{\"isExcluded\":false,\"includeDescendants\":false,\"includeMapped\":false}]}},{\"id\":1,\"name\":\"Hypertension\",\"expression\":{\"items\":[{\"isExcluded\":false,\"includeDescendants\":false,\"includeMapped\":false}]}},{\"id\":2,\"name\":\"AtrialFibrilation\",\"expression\":{\"items\":[{\"isExcluded\":false,\"includeDescendants\":false,\"includeMapped\":false}]}},{\"id\":3,\"name\":\"StatinVS\",\"expression\":{\"items\":[{\"isExcluded\":false,\"includeDescendants\":false,\"includeMapped\":false}]}},{\"id\":4,\"name\":\"ColonoscopyVS\",\"expression\":{\"items\":[{\"isExcluded\":false,\"includeDescendants\":false,\"includeMapped\":false}]}}],\"QualifiedLimit\":{\"Type\":\"First\"},\"ExpressionLimit\":{\"Type\":\"First\"},\"InclusionRules\":[{\"name\":\"PhenotypeCase\",\"description\":\"PhenotypeCase\",\"expression\":{\"Type\":\"ALL\",\"CriteriaList\":[{\"Criteria\":{\"ConditionOccurrence\":{\"CodesetId\":0,\"ConditionTypeExclude\":false}},\"StartWindow\":{\"Start\":{\"Coeff\":-1},\"End\":{\"Coeff\":1}},\"Occurrence\":{\"Type\":2,\"Count\":1,\"IsDistinct\":false},\"RestrictVisit\":false}],\"DemographicCriteriaList\":[],\"Groups\":[{\"Type\":\"ANY\",\"CriteriaList\":[{\"Criteria\":{\"ConditionOccurrence\":{\"CodesetId\":1,\"ConditionTypeExclude\":false}},\"StartWindow\":{\"Start\":{\"Coeff\":-1},\"End\":{\"Coeff\":1}},\"Occurrence\":{\"Type\":2,\"Count\":1,\"IsDistinct\":false},\"RestrictVisit\":false},{\"Criteria\":{\"ConditionOccurrence\":{\"CodesetId\":2,\"ConditionTypeExclude\":false}},\"StartWindow\":{\"Start\":{\"Coeff\":-1},\"End\":{\"Coeff\":1}},\"Occurrence\":{\"Type\":2,\"Count\":1,\"IsDistinct\":false},\"RestrictVisit\":false}],\"DemographicCriteriaList\":[],\"Groups\":[]}]}}],\"CollapseSettings\":{\"CollapseType\":\"ERA\",\"EraPad\":0}}"
```

Comment:
Why did this file change?

Reply:
Okay, I guess it's because the new valuesets.

Reply:
I integrated changes into the large testing CQL file, which introduced some new value sets.  They started appearing in this output.